### PR TITLE
[v7r2] renames of column names to avoid reserved keywords

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Core
FIX: use logging.warning in favor of deprecated logging.warn

*Framework
CHANGE: renames of column names to avoid reserved keywords

ENDRELEASENOTES

### CODE REVIEW

**Review by chrisburr — DISMISSED** [review]

**Comment by TaykYoku** [comment]
@fstagni I submitted a #4958 PR today, that already contains some changes to this PR:
```
src/DIRAC/ConfigurationSystem/scripts/dirac_admin_add_site.py 
src/DIRAC/ConfigurationSystem/scripts/dirac_admin_sort_cs_sites.py 
src/DIRAC/ConfigurationSystem/scripts/dirac_configuration_dump_local_cache.py 
src/DIRAC/Core/scripts/dirac_agent.py 
src/DIRAC/Core/scripts/dirac_executor.py 
src/DIRAC/Core/scripts/dirac_install_extension.py 
src/DIRAC/Core/scripts/dirac_service.py
```

**Comment by fstagni** [comment]
Yes I saw that, and also started going through it. We can merge yours quickly and then I can rebase this small one of mine.

**Comment by atsareg** [comment]
Please resolve conflicts due to thee merged #4958

**Comment by andresailer** [comment]
The changed column names need to be mentioned here https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v7r2 right?

**Review by andresailer — DISMISSED** [review]

**Comment by fstagni** [comment]
> The changed column names need to be mentioned here https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v7r2 right?

Yes, done.

**Review by andresailer — APPROVED** [review]

File: src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py [hunk 1]
```diff
@@ -586,10 +587,11 @@ def removeComponentOptionsFromCS(self, system, component, mySetup=None):
       else:
         isRenamed = True
 
-      result = self.monitoringClient.getInstallations({'UnInstallationTime': None},
-                                                      {'System': system,
-                                                       'Module': installation['Component']['Module']},
-                                                      {}, True)
+      result = self.monitoringClient.getInstallations(
+          {'UnInstallationTime': None},
+          {'System': system, 'DIRACModule': installation['Component']['Module']},
```
Comment (chaen) [inline src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py:592]: @fstagni should it be `DIRACSystem` here ?

Comment (fstagni) [reply src/DIRAC/FrameworkSystem/Client/ComponentInstaller.py:592]: Thanks, fixed in my open PR.

**Pull request merged**